source/base/ByteBuffer.ooc

```diff
@@ -28,26 +28,29 @@ ByteBuffer: class {
 	size ::= this _size
 	_referenceCount: ReferenceCounter
 	referenceCount ::= this _referenceCount
-	init: func (=_pointer, =_size) {
+	_owner: Bool
```

Comment:
This is a strange name for a `Bool`. I can't think of a better one that doesn't also sound silly, though. `_ownsItself`, `_hasOwner`, `_isHisOwnMaster`, etc. are all obviously bad, too.